app/views/provider/admin/backend_apis/metrics/index.html.slim

```diff
@@ -10,99 +10,87 @@ section.Section.u-toggleableBySettingsBox data-state="open"
-            })
-            $(document).on('click', 'a[data-behavior~=open-mapping-rules]', function(event){
-              // Unfold the mapping section of the integration page by
-              // setting the cookie value that controls it
-              $.cookie( 'mapping-rules', true, {expires: 30, path: '/'});
-            });
+        javascript:
+          document.addEventListener('DOMContentLoaded', function () {
+            showBubble(#{json bubbles.take(1).as_json})
+          })
+          $(document).on('click', 'a[data-behavior~=open-mapping-rules]', function(event){
+            // Unfold the mapping section of the integration page by
+            // setting the cookie value that controls it
+            $.cookie( 'mapping-rules', true, {expires: 30, path: '/'});
+          });
 
-        - else
-          tr
-            td colspan=(show_mappings? ? '6' : '5')
-              = t('provider.admin.messages.no_methods')
+      - else
+        tr
+          td colspan='6'
+            = t('provider.admin.messages.no_methods')
 
 section.Section.u-toggleableBySettingsBox data-state="open"
   h2 Metrics
   p
     ' <em>Hits</em> is the built-in metric to which all methods report. Additional top-level metrics can be added here in order to track other usage that shouldn't increase the hit count.
-    - if show_mappings?
-      ' A metric needs to be mapped to one or more URL patterns in the
-      => link_to 'Mapping Rules', provider_admin_backend_api_mapping_rules_path(@backend_api)
-      ' of this Backend API
-    - else
-      ' Make sure to call these metrics from your code base so specific calls to your API up the count of specific metrics.
+    ' A metric can be mapped to one or more URL patterns in the
+    => link_to 'Mapping Rules', provider_admin_backend_api_mapping_rules_path(@backend_api)
+    ' of this Backend API
 
-  - cache ['v1', 'metrics', @backend_api] do
-    table#metrics.data class=(show_mappings? ? 'u-sixEqualColumns' : 'u-fiveEqualColumns')
-      thead
-        tr
-          th Metric
-          th System Name
-          th Unit
-          th Description
-          - if show_mappings?
-            th Mapped
-          th.actions
-            = link_to 'New metric',
-                      new_provider_admin_backend_api_metric_path(@backend_api),
-                      class:'action add'
-
-      tbody
-        - if @backend_api.top_level_metrics.any?
-          - @backend_api.top_level_metrics.each do |metric|
-            tr
-              td
-                = link_to metric.friendly_name, edit_provider_admin_backend_api_metric_path(@backend_api, metric)
-              td = metric.system_name
-              td = metric.unit
-              td = metric.description
-              - if show_mappings?
-                td
-                  = render 'link_to_mapping_rules', metric: metric
-              td.actions
+  table#metrics.data class='u-sixEqualColumns'
```

Comment:
Would it make more sense to place Metrics above Methods and not the other way around? Since Metrics are top-level and Methods, children.

Reply:
If we do, it should be done both for products and for backend APIs. I'd say out of scope for this PR.

Reply:
OK